category/interfaces/graphql/resolvers.go

```diff
@@ -55,7 +56,6 @@ func (r *CommerceCategoryQueryResolver) CommerceCategory(
 			PageSize:         request.PageSize,
 			Page:             request.Page,
 			SortBy:           request.SortBy,
-			SortDirection:    request.SortDirection,
```

Comment:
We resolved it, in the first run, because it aligned with the UX to that point. Now we came across the implementation / usage of the sort options data and saw some disadvantages in it. The extra logics for grouping different directions lead to extra logic in the frontend service as well.
- There was a need for configuration of available sort directions for specific sort options, because just some of them really had an option for asc and desc.
- The label that comes from our data provider could not be used, so we would need to translate the options in the frontend service, again.
- The logics in flamingo does not target all options, so sort options that are not asc and desc, or does not follow that naming convention, does not need the direction to be passed at all, what again lead to logic in the frontend service.

That's why we decided to go for a different solution, that streamlines a lot more with the other implementations for data we get via GraphQL. Using the sort options right as we get them, with their own managed label and passing their real (full) codes with the sortBy query parameter, makes us get rid of all the configuration, translation and logics. The UX we go for now is to simply show 2 options for, e.g. `Price high to low` and `Price low to high` without any grouping or marking a single `Price` option to look like asc or desc.
In that favor we removed the sort direction from the GraphQL resolver again.

Reply:
Ok thanks - so the sort gets something like a "sortdirective" that does not need to match with a fieldname?

So if that makes more sense there are a couple of follow ups - like:
- change the SortFilter (currently has field and direction - can be renamed to sortdirective)
- rename "field" in shema? https://github.com/i-love-flamingo/flamingo-commerce/blob/2c431c8d976d9addcbd0bdb84a383fac470d1f24/search/interfaces/graphql/schema.graphql#L29  can be renamed to no

Reply:
I see your points.
The `SortBy` was a naming that - i thought - fitted well, even without the direction, as it nicely describes what happens and did not state something like `SortByField`. But this could be renamed, as long as we don't break existing projects :)
The `field` could be renamed as well (to `directive` ?).
Let's PM about the w's :)